<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>427</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>900</width>
    <height>427</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>900</width>
    <height>427</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Библиотека</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>about_authors/5.png</normaloff>about_authors/5.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #2f3136;
color: #bbbbbb;</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLineEdit" name="lineEdit_search">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #36393f;
border-style: outset;
border-width: 2px;
border-radius: 10px;
border-color: beige;
</string>
    </property>
    <property name="text">
     <string>Поиск автора</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_search">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>10</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #2f3136;
border-style: outset;
border-width: 2px;
border-radius: 10px;
border-color: beige;
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QGroupBox" name="books_Box">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>300</y>
      <width>621</width>
      <height>121</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="title">
     <string>Книги автора</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="autors_Box">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>251</width>
      <height>371</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="title">
     <string>Список авторов</string>
    </property>
    <widget class="QListWidget" name="listWidget_authors">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>231</width>
       <height>341</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: #36393f;
border-style: outset;
border-width: 2px;
border-radius: 10px;
border-color: beige;</string>
     </property>
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="about_author_Box">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>0</y>
      <width>301</width>
      <height>301</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="title">
     <string>Об авторе</string>
    </property>
    <widget class="QTextEdit" name="textEdit_author">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>191</width>
       <height>271</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: #36393f;
border-style: outset;
border-width: 2px;
border-radius: 10px;
border-color: beige;</string>
     </property>
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="autoFormatting">
      <set>QTextEdit::AutoAll</set>
     </property>
     <property name="undoRedoEnabled">
      <bool>true</bool>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="photo_author_label">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>20</y>
       <width>81</width>
       <height>131</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-style: outset;
border-width: 2px;
border-radius: 10px;
border-color: beige;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="about_book_Box">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>0</y>
      <width>311</width>
      <height>301</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="title">
     <string>О книге</string>
    </property>
    <widget class="QTextEdit" name="textEdit_aboutBook">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>291</width>
       <height>271</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: #36393f;
border-style: outset;
border-width: 2px;
border-radius: 10px;
border-color: beige;
</string>
     </property>
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="undoRedoEnabled">
      <bool>false</bool>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
